package.json

```diff
@@ -4,8 +4,10 @@
   "description": "Wrapper to run nemo/mocha suites",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && ./bin/nemo-runner -B test -G @suite1",
-    "test:parallel": "./bin/nemo-runner -B test -G @suite1,@suite2,@suite3,@suite4,@suitex -P chrome,firefox",
+    "test": "npm run lint && npm run nemo",
```

Comment:
It'd be nice to have this run all the nemo scripts (debug and parallel)- or at least have travis run them.

Reply:
True. Good enhancement.